### DISCUSSION TITLE
[NOVA TRILHA] Produtos Digitais

### DISCUSSION THREAD

**Discussion by guilhermenstorti** [codaqui/institucional#82] [Trilhas de Ensino]
Trilha destinada ao ensino de gestão de produtos digitais e formação de Product Managers.

### Tópicos sugeridos:
**Módulo 1**
  1.1) Introdução e fundamentos de Product Management 
  1.2) Habilidadeds esperadas de um Product Manager
  1.3) Estrutura de times de produto
  1.4) Papel de um Head (ou VP) de produto

**Módulo 2**
  2.1) O conceito de valioso, factível, viável e usável 
  2.2) Modelos de Negócio e propostas de valor
  2.3) Market Fit e Unit Economics
  2.4) Descobrindo necessidades dos usuários

**Módulo 3**
  3.1) Criar a estratégia e visão do produto
  3.2) Criar o roadmap de produto
  3.3) Técnicas de priorização
  3.4) Release planning
  3.5) Métricas de negócio que impactam o produto
  3.6) Métricas de produto
  3.7) Usando growth hacking para atingimento de metas
  3.8) Frameworks de métricas aplicados em mobile growth

**Módulo 4**
  4.1) Fundamentos de UX
  4.2) Como um PM e UX trabalham juntos
  4.3) Explorar problemas e descobrir oportunidades
  4.4) Ideação e definição da solução
  4.5) Validação, iteração e dicas finais
  4.6) Customer Journey

**Módulo 5**
  5.1) Especificações de requerimentos e rollout
  5.2) Metodologias ágeis
  5.3) Como usar dados para tomar decisões
  5.4) SQL para análise de dados
  5.5) Testes A/B
  5.6) Como gerencial stakeholders
  5.7) Ferramentas úteis

Bibliografias interessantes:

**Comment by endersonmenezes** [comment]
@Divcarvalho olha que bacana! 
Muito obrigado Storti, eu acho que temos um bom começo aqui! =]
Pensando que trabalharemos isso com pessoas de 12 a 18 anos. O que você consideraria opcional nesta trilha? Ou todos os tópicos você acha que dá pra trabalhar ele com qualquer idade, flexibilizando o conteúdo.

PS: @guilhermenstorti Acredito que isso vai levar tempo, então para contabilizarmos o ponto disso para o Hacktoberfest você pode criar apenas o arquivo da trilha em branco com o nome: `produtos-digitais.md`?

**Reply by guilhermenstorti** [reply]
Vou deixar melhor descrito os tópicos que acho essenciais, optativos ou com conteúdo flexibilizado baseado na idade. Por hora, já fiz o pr do arquivo em branco!

**Reply by endersonmenezes** [reply]
PR: https://github.com/codaqui/institucional/pull/83

**Comment by endersonmenezes** [comment]
Pesquisa para próximo encontro @kamisinha @ArtumosGOC @LuizFernando-TC @SOPHI-A9 

Modulo1: @kamisinha 
Modulo2: @SOPHI-A9 
Modulo3: @LuizFernando-TC 
Modulo4: @ArtumosGOC 

## Módulo 5

5.1) Especificações de requerimentos e rollout
- Link 1
- Link 2
- Video

5.2) Metodologias ágeis
- Link 1
- Link 2
- Video

5.3) Como usar dados para tomar decisões
- Link 1
- Link 2
- Video


5.4) SQL para análise de dados
- Link 1
- Link 2
- Video


5.5) Testes A/B
- Link 1
- Link 2
- Video

5.6) Como gerencial stakeholders
- Link 1
- Link 2
- Video

5.7) Ferramentas úteis
- Link 1
- Link 2
- Video

**Comment by endersonmenezes** [comment]
@guilhermenstorti pode nos ajudar a criar a trilha agora seguindo o novo modelo?

**Reply by guilhermenstorti** [reply]
@endersonmenezes posso sim, sem problemas. Vou ver se consigo trabalhar melhor isso durante o final de semana

**Reply by endersonmenezes** [reply]
Beleza, a gente melhorou bastante o sistema e a interface de estudo, então agora conseguiremos criar algo bem legal. Eu também linkei aqui por que as buscas sobre isso em um sistema de monitoramente nosso cresceu, pessoas estão buscando coisas sobre produtos digitais e pode ser uma chance pra gente.
